lambda/layer/main.tf

```diff
@@ -1,6 +1,14 @@
+locals {
+  package_s3_match = var.package_s3 != null ? regex(
```

Comment:
Does it work with key of a nested object? Probably it does :) Do you prefer this package_s3 approach rather than package_s3_bucket and package_s3_object instead?

Reply:
converted `package_s3` to a `{bucket, key}` object in 76c00b4